ceph.spec.in

```diff
@@ -536,6 +577,21 @@ librbd1 or libcephfs1 instead.
 
 %endif
 
+%package compat
```

Comment:
Those will need to stay for now (we could probably drop them after Jewel, since it's been so long, but let's not touch them now.) You can just drop this one ceph-compat package.